docs/superpowers/plans/2026-08-27-builder-credentials-export.md

```
@@ -0,0 +1,958 @@
+- Requires `--app-id <APP_ID>` (the compatibility alias `--appId` is also accepted).
+- Requires exactly one of:
+  - `--raw`: prints only the exact stored value to stdout, with no trailing newline.
+  - `--file <PATH>`: creates a new `0600` file and never overwrites an existing path.
+- Use `--decode-base64` with `--file` to write decoded bytes; otherwise Base64 is saved as stored.
+- Use `--local` or `--global` when the app exists in both stores.
+- Use `--platform ios|android` when platform values are ambiguous.
+
+```bash
+npx @capgo/cli@latest build credentials export ANDROID_KEYSTORE_FILE \
+  --app-id com.example.app --platform android --file ./release.keystore --decode-base64
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)